src/pages/home/report/ReportActionsView.js

```diff
@@ -245,6 +248,55 @@ function ReportActionsView(props) {
         }
     }, [hasCachedActions]);
 
+    // When we offline before opening a money request report,
+    // the total of the report and sometimes the money request aren't displayed because these actions aren't returned until `OpenReport` API is complete.
+    // We generate a fake created action here if it doesn't exist to display the total whenever possible because the total just depends on report data
+    // and we also generate a money request action if there is no such action to display at least one money request action to match the total data.
+    // For more details: https://github.com/Expensify/App/issues/26424#issuecomment-1869154198
+    const reportActionsToDisplay = useMemo(() => {
+        if (!ReportUtils.isMoneyRequestReport(props.report) || !_.size(props.reportActions)) {
+            return props.reportActions;
+        }
+
+        const actions = [...props.reportActions];
+
+        if (!ReportActionsUtils.isCreatedAction(_.last(props.reportActions))) {
+            const optimisticCreatedAction = ReportUtils.buildOptimisticCreatedReportAction(
+                props.report.ownerAccountID,
+                DateUtils.subtractMillisecondsFromDateTime(_.last(props.reportActions).created, 1),
+            );
+            actions.push(optimisticCreatedAction);
+        }
+
+        if (
+            props.report.total &&
+            _.filter(
+                props.reportActions,
+                (action) => action.actionName === CONST.REPORT.ACTIONS.TYPE.IOU && action.originalMessage && action.originalMessage.type === CONST.IOU.REPORT_ACTION_TYPE.CREATE,
+            ).length === 0
+        ) {
+            const optimisticIOUAction = ReportUtils.buildOptimisticIOUReportAction(
```

Comment:
I tried to test this case locally by removing the if checks, the skeleton comes. But it is not animating like other skeletons do.



https://github.com/Expensify/App/assets/46707890/73434c0b-0b31-4dff-b7ff-b21538e652af

Reply:
@dukenv0307 

>The reason is MoneyRequestSkeletonView has backgroundColor and foregroundColor are the same, no animation appears.

Is it even previously we didn't have animation where the money request skeleton was display? 🤔 
I think I remember seeing animation, not sure though